i18n-embed/src/fluent.rs

```diff
@@ -93,6 +93,16 @@ impl FluentLanguageLoader {
         }
     }
 
+    /// The languages associated with each actual loaded language bundle.
+    pub fn current_languages(&self) -> Vec<unic_langid::LanguageIdentifier> {
```

Comment:
This could be called something else. I borrowed the name from the Trait `LanguageRequester`.

Reply:
sounds fine to me!